.github/workflows/update-bootstrap.yml

```diff
@@ -0,0 +1,44 @@
+name: Update Bootstrap nodes
+
+on:
+  release:
+    types: [released]
+
+jobs:
+  update-bootstrap-nodes:
+    name: Update Bootstrap nodes
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        environment: [bootstrap-devqa, bootstrap-tnet-mainnet]
+      max-parallel: 1
+    steps:
+      - uses: actions/checkout@v4
+      - name: Get Release Version from Cargo.toml
+        id: cargo_version
+        run: |
+          VERSION=$(grep '^version' Cargo.toml | sed -E 's/version = "(.*)"/\1/')
+          echo "VERSION=$VERSION" >> $GITHUB_ENV
+      - name: Checkout Ansible Playbook
+        uses: actions/checkout@v4
+        with:
+          repository: ceramicstudio/simpledeploy
+          path: simpledeploy
+      - name: Add SSH key
+        run: |
+          echo "${{ secrets.ANSIBLE_SSH_KEY }}" > ~/.ssh/ansible_ssh_key
```

Comment:
Did you say that the keys are different for the two envs? If so, should the secret name for each env come from the strategy as well?